server/http.go

```diff
@@ -26,13 +26,27 @@ import (
 const (
 	defaultMeetingTopic      = "Zoom Meeting"
 	zoomOAuthUserStateLength = 4
+	settingDataError         = "something went wrong while getting settings data"
+	askForPMIMeeting         = "Would you like to use your personal meeting ID?"
+	apiToUpdatePMI           = "/plugins/%s/api/v1/updatePMI"
+	apiToAskForPMI           = "/plugins/%s/api/v1/askPMI"
```

Comment:
Seems off that this has a constant but not the regular `/api/v1/askPMI` string below